analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -48,6 +48,10 @@ import org.ossreviewtoolkit.spdx.toSpdx
 
 private const val DEFAULT_SCOPE_NAME = "default"
 
+fun SpdxPackage.getAuthorsFromCopyrightText(): SortedSet<String> =
+    Regex("(?=copyright \\(c\\))", RegexOption.IGNORE_CASE).split(copyrightText).filter { it.isNotEmpty() }
```

Comment:
For other package managers, currently only author information is extracted without a copyright prefix (an many of them do not support this information). To be in-line with these implementations, should this prefix be dropped? It would, however, be a waste to throw this information away when it is present.

Reply:
Was there a special reason why exactly this prefix was chosen to split copyright texts? There are other parts of ORT, which use a rather complex logic for dealing with copyrights, for instance the `CopyrightStatementsProcessor` class. Compared with these approaches, the hard-coded constant may be over-simplified.

Reply:
Thanks for pointing me to the `CopyrightStatementsProcessor` I wasn't aware of it. I'll rethink the implementation

Reply:
Actually, I also believe we should rethink which field to use here. It does not really make sense to me to strip copyright prefixes from `copyrightText` here in order to make an `author` out of it, and later we'll probably turn authors again into copyright holders by adding a copyright prefix.

So, for the sake of extracting *author* information, I believe we should use `originator: Person:` here, and, in contrast to what I said earlier internally, probably also use `originator: Organization:`, even if that field is also already used as a namespace. Maybe we should *not* use `originator: Organization:` as a namespace after all...

Reply:
@tsteenbe what's your though on this?

Reply:
I initially used `originator`. Additionally to your mentioned internal discussion where we though we didn't want to use `originator` twice, I feel like the `copyrightText` supplies more information since originator can always only be a single value, one originator and `copyrightText` can hold multiple copyrights.

Reply:
@tsteenbe @sschuberth any new opinions on this?

Reply:
@tsteenbe @sschuberth any new opinions on which field to use for the new author field? 
`originator`  or `copyrightText`? (see discussion above)

Reply:
I gave this some though again after the last dev meeting. To me, we should:
- Use `originator` as the "author", and make use of the existing new mechanism to deduce copyright holders from authors.
- In the future, treat `copyrightText` as yet another new potential source of copyright holder information in the context of solving https://github.com/oss-review-toolkit/ort/issues/3780.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/4527.